pkg/scheduler/scheduler.go

```diff
@@ -250,6 +255,7 @@ func (s *Scheduler) nominate(ctx context.Context, workloads []workload.Info, sna
 			e.inadmissibleMsg = err.Error()
 		} else {
 			e.assignment = flavorassigner.AssignFlavors(log, &e.Info, snap.ResourceFlavors, cq)
+			e.preemptionTargets = s.preemptor.GetTargets(e.Info, e.assignment, &snap)
```

Comment:
if it's Fit, could we skip `GetTargets`?

Reply:
sure